core/transport/src/main/proto/grpc/meta_master.proto

```diff
@@ -244,6 +246,12 @@ message MasterHeartbeatPResponse {
   optional MetaCommand command = 1;
 }
 
+message UpdateConfigPRequest {
+  map<string, string> properties = 1;
```

Comment:
```suggestion
  optional map<string, string> properties = 1;
```

good practise to keep every field optional for future reference

Reply:
Sorry, Map fields cannot have labels